index.html

```diff
@@ -1,81 +1,105 @@
 <html>
 <head>
-<title>Wolfbrains</title>
-<style type="text/css">
-  img { border-radius: 15px; width: 25px; height: 25px; display: inline;}
-</style>
-<script type="text/javascript" src="yaml.min.js"></script>
-<script type="text/javascript" src="gravatar.js"></script>
-<script type="text/javascript">
-  document.addEventListener("DOMContentLoaded", function(event) {
-    var dataStore = {};
-    var currentYear = new Date().getFullYear();
-    var theBrains = document.getElementById('wolfbrains');
-    YAML.load('http://ckdake.github.io/wolfbrains/data.yml', function(result)
-    {
-        nativeObject = result;
-        nativeObject.people.forEach(function(peopleEntry) {
-          peopleEntry.jobs.forEach(function(jobEntry) {
-            var company = jobEntry['company'];
-            var startYear = jobEntry['start_day'].getFullYear();
-            var endYear = currentYear;
-            if (jobEntry['end_day'] != null) {
-                endYear = jobEntry['end_day'].getFullYear();
-            }
+    <title>Wolfbrains</title>
+    <style type="text/css">
+        img {
+            border-radius: 15px;
+            width: 25px;
+            height: 25px;
+            display: inline;
+        }
+    </style>
+    <script type="text/javascript" src="jquery-1.11.3.min.js"></script>
+    <script type="text/javascript" src="gravatar.js"></script>
+    <script type="text/javascript">
+        document.addEventListener("DOMContentLoaded", function (event) {
+            var dataStore = {};
+            var currentYear = new Date().getFullYear();
+            var theBrains = document.getElementById('wolfbrains');
 
-            for (year = startYear; year <= endYear; year++) {
-              if (!(dataStore.hasOwnProperty(year))) { dataStore[year] = {} };
-              if (!(dataStore[year].hasOwnProperty(company))) { dataStore[year][company] = [] };
-              dataStore[year][company].push(peopleEntry);
+            var fullYearFromDateString = function(string) {
+                var date = new Date(string);
+                return date.getFullYear();
             }
-          });
-          if (peopleEntry.education) {
-            peopleEntry.education.forEach(function(educationEntry) {
-              var institution = educationEntry['institution'];
 
-              if (educationEntry['start_day'] != null) {
-                var startYear = educationEntry['start_day'].getFullYear();
-              }
 
-              var endYear = currentYear;
-              if (educationEntry['end_day'] != null) {
-                  endYear = educationEntry['end_day'].getFullYear();
-              }
+            jQuery.getJSON('http://localhost:8000/data.json', function (result) {
```

Comment:
Lol. I was so proud of myself for figuring this out I didn't go back and look over my work =(